tests/examples/java_examples_test.go

```diff
@@ -133,6 +133,14 @@ func TestExamples(t *testing.T) {
 		integration.ProgramTest(t, &test)
 	})
 
+	t.Run("kubernetes", func(t *testing.T) {
+		t.SkipNow()
```

Comment:
Do we want to skip this? Or is this still WIP? I'm guessing this doesn't exercise the codepath right now, is that correct? As you say, we'd need some better testing infrastructure to make it possible?

Reply:
Am skipping until the kubernetes provider has been updated with a new release of pulumi-java. It is a catch-22 that this PR is needed first.  Of course I would have preferred a self-contained example but I was unable to find a way to exercise the relevant code path without  the use of an MLC.  Maybe the 'echo' provider in the "pulumi" submodule could be used, but I don't have time to figure that out.